server/aesdsocket-start-stop

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
@adna4700 make sure to remove your aesdsocket application from your repo and add to .gititnore.